circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Maybe [`mdimport /Applications/Xcode*`](http://osxdaily.com/2012/02/02/reindex-spotlight-from-the-command-line/) and retry.